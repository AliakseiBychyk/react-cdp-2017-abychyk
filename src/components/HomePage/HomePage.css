.home {
  text-align: center;
  padding: 0;
}