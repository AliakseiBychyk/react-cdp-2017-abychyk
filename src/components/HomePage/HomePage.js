import React from 'react'
import SearchHeader from '../SearchHeader/SearchHeader'
import styles from './HomePage.css'

const HomePage = () => (
  <div className={styles.home}>
    <SearchHeader />
  </div>
)

export default HomePage