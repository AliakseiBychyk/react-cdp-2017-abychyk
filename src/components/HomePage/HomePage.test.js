import React from 'react'
import { shallow } from 'enzyme'
import HomePage from './HomePage'

test('should render HomePage correctly', () => {
  const redirectTo = jest.fn()
  const wrapper = shallow(<HomePage redirectTo={redirectTo} />)
  expect(wrapper).toMatchSnapshot()
})
