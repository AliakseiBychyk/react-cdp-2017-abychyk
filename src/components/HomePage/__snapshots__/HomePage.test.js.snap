// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<div
  className="home"
>
  <SearchHeader
    criterion="person"
    defaultQuery="Quentin Tarantino"
    query=""
    redirectTo={[Function]}
  />
</div>
`;
